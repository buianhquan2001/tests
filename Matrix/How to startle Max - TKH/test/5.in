34
111
91
67
93
88
76
108
68
102
115
93
102
120
117
93
63
71
65
75
108
86
82
79
86
77
117
110
84
74
63
61
113
103
103
