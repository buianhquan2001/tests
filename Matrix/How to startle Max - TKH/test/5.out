466486836
672789790
718461253
482361088
142382092
376671503
419333973
599807473
635707616
721568908
482361088
635707616
951234175
273207878
482361088
881779447
51288244
562992784
255840229
419333973
980141720
639909956
471084001
980141720
216191719
273207878
170399885
130003129
413021001
881779447
312450027
642654783
981043477
981043477
